pkg/cmd/testdata/keda_autoscaler.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: example-app
+spec:
+  image: "ghcr.io/foo/example-app:v0.1.0"
+  executor: containerd-shim-spin
+  enableAutoscaling: true
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: example-app-autoscaler
+spec:
+  scaleTargetRef:
+	apiVersion: apps/v1
+	kind: Deployment
```

Comment:
these lines look incorrectly indented.

Reply:
Good catch. I’ll go fix that as well.

Reply:
Looks like it's due to the source file using tabs and spaces for indendation. Will fix

Reply:
Thank you Matt.